### ISSUE TITLE
other.test_legalize_js_ffi fails to get BINARYEN automatically

### ISSUE THREAD

**Issue by saschanaz** [kripken/emscripten#5789]
default.test_binaryen gets it automatically but other.test_legalize_js_ffi does not get it.

```
WARNING:root:use EM_ALL_ENGINES=1 in the env to run against all JS engines, which is slower but provides more coverage
Test suites:
['test_other']
Running test_other: (1 tests)
test_legalize_js_ffi (test_other.other) ... ['-s', 'LEGALIZE_JS_FFI=1', '-s', 'SIDE_MODULE=1', '-O2']
C:\Users\sasch\AppData\Local\Programs\Python\Python36\python.exe C:\Users\sasch\Documents\GitHub\emsdk\emscripten\incoming\emcc C:\Users\sasch\Documents\GitHub\emsdk\emscripten\incoming\tests\other\ffi.c -s WASM=1 -g -o a.out.js -s LEGALIZE_JS_FFI=1 -s SIDE_MODULE=1 -O2
WARNING:root:BINARYEN_ASYNC_COMPILATION disabled due to user options. This will reduce performance and compatibility (some browsers limit synchronous compilation), see https://github.com/kripken/emscripten/wiki/WebAssembly#codegen-effects
Traceback (most recent call last):
  File "C:\Users\sasch\Documents\GitHub\emsdk\emscripten\incoming\emcc", line 13, in <module>
    emcc.run()
  File "C:\Users\sasch\Documents\GitHub\emsdk\emscripten\incoming\emcc.py", line 1834, in run
    wasm_text_target, misc_temp_files, optimizer)
  File "C:\Users\sasch\Documents\GitHub\emsdk\emscripten\incoming\emcc.py", line 2304, in do_binaryen
    subprocess.check_call(cmd)
  File "C:\Users\sasch\AppData\Local\Programs\Python\Python36\lib\subprocess.py", line 286, in check_call
    retcode = call(*popenargs, **kwargs)
  File "C:\Users\sasch\AppData\Local\Programs\Python\Python36\lib\subprocess.py", line 267, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\sasch\AppData\Local\Programs\Python\Python36\lib\subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "C:\Users\sasch\AppData\Local\Programs\Python\Python36\lib\subprocess.py", line 997, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
FAIL

======================================================================
FAIL: test_legalize_js_ffi (test_other.other)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\sasch\Documents\GitHub\emsdk\emscripten\incoming\tests\test_other.py", line 7789, in test_legalize_js_ffi
    assert proc.returncode == 0
AssertionError

----------------------------------------------------------------------
Ran 1 test in 5.282s

FAILED (failures=1)
```

What would be the cause?

**Comment by kripken** [comment]
If that is the first wasm test to run in that VM, then it might fail because it uses SIDE_MODULE - which should be fixed by #5763. It's waiting on someone to read through it and say it looks ok - if you have time, that would be appreciated.

**Comment by saschanaz** [comment]
Good, if this is fixed then we can safely make BINARYEN_ROOT an empty string on `.emscripten` file so that it will always get the latest bits.

**Comment by saschanaz** [comment]
Fixed by #5763.
